packages/server/test/support/helpers/e2e.js

```diff
@@ -646,7 +612,11 @@ const e2e = {
         const ostd = options.onStdout
 
         if (ostd) {
-          stdout = ostd(stdout)
+          const newStdout = ostd(stdout)
```

Comment:
worth mentioning the decision/choice here to allow `onStdout` to not return anything, so we only reset it when a string is returned

Reply:
addressed e2e.it comment